src/main/java/org/grizz/exception/MaxNumberOfPlanetException.java

```diff
@@ -0,0 +1,5 @@
+package org.grizz.exception;
+
+public class MaxNumberOfPlanetException extends RuntimeException {
```

Comment:
I don't think it is time for validating such thing. For now we will have only 1 planet per user, later when we will have Colonization Ships we will make such limit. Not earlier